bin/picca_metal_dmat.py

```diff
@@ -416,6 +422,16 @@ def main(cmdargs):
         for index2, abs_igm2 in enumerate(abs_igm_2[index0:]):
             if index1 == 0 and index2 == 0:
                 continue
+
+            if args.fast_metals:
+                default_si_metals = ['SiIII(1207)','SiII(1190)','SiII(1193)','SiII(1260)']
+                if (abs_igm1=='LYA')&(abs_igm2 in default_si_metals):
+                    pass
+                elif abs_igm1==abs_igm2=='CIV(eff)':
+                    pass
+                else:
```

Comment:
why not just put one condition and then the `continue`?  

something like:
```
if not (abs_igm1 == "Lya" and abs_igm2 in default_si_metals) and not (abs_igm1 == "CIV(eff)" and abs_igm1 == abs_igm2):
    continue
```

Reply:
changed to this